### ISSUE TITLE
Embedding some views in an Edit view while using UUID as ID fails 

### ISSUE THREAD

**Issue by iron3oxide** [saltcorn/saltcorn#2698]
To preface this, I am using UUIDs as IDs in all my tables. I have a view in which I'd like to embed either a Gantt chart or a calendar to display some timeseries-ish data. Both the saltcorn-gantt and the fullcalendar module give the following error:
```
error: invalid input syntax for type uuid: "NaN"
    at /home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/pg-pool/index.js:45:11
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at Table.getJoinedRows (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/models/table.ts:3504:17)
    at Object.run (/home/saltcorn/.local/share/saltcorn-plugins/plugins_folder/@saltcorn/fullcalendar/index.js:639:16)
    at View.run (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/models/view.ts:471:14)
    at Object.view (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/base-plugin/viewtemplates/edit.js:872:9)
    at go (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/models/layout.ts:61:44)
    at go (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/models/layout.ts:71:49)
    at go (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/models/layout.ts:75:40)
    at traverse (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/models/layout.ts:83:3)
    at transformForm (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/base-plugin/viewtemplates/edit.js:591:3)
    at render (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/base-plugin/viewtemplates/edit.js:1032:3)
    at editQuery (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/base-plugin/viewtemplates/edit.js:2037:14)
    at Object.run (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/base-plugin/viewtemplates/edit.js:511:10)
    at View.run (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/models/view.ts:471:14)
    at Object.run (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/base-plugin/viewtemplates/listshowlist.js:211:13)
    at View.run (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/models/view.ts:471:14)
    at Object.view (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/models/page.ts:266:11)
    at go (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/models/layout.ts:61:44)
    at go (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/models/layout.ts:75:40)
    at traverse (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/models/layout.ts:83:3)
    at Page.run (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/data/models/page.ts:234:5)
```
The svelte-gantt plugin works but is not really suitable for me in its current form. I have tried presetting all UUID fields (there are two foreign keys in the table of the view) with an extra state formula, but to no avail.

**Comment by glutamate** [comment]
@iron3oxide you seem to get better error messages than me! which version of node are you running?

**Comment by iron3oxide** [comment]
@glutamate Should be v18.19.0, however I should note that in most scenarios, I only got the first line as a red toast. I think it was only when using a view link button instead that I got this detailed error after clicking on it.

**Comment by glutamate** [comment]
Fullcalendar fixed in that module v 0.4.2

**Comment by glutamate** [comment]
I have submitted a PR for saltcorn-gantt (developed by @AleksandrSokolov )
